### DISCUSSION TITLE
1panel无法添加postgresql的远程服务器连接

### DISCUSSION THREAD

**Discussion by jjscx** [1Panel-dev/1Panel#7093] [Q&A]
### 联系方式

jjscx@outlook.com

### 1Panel 版本

v1.10.20-lts

### 问题描述

1panel提供的远程数据库无法连接本机docker部署的postgresql   非个例

### 重现步骤

1. 安装docker 
2. docker内安装postgresql
3. 安装1panel
4. 1panel尝试连接postgresql
5. 失败

### 相关日志输出

```shell
postgresql无任何日志打印  其他工具可以正常连接
```

### 附加信息

https://bbs.fit2cloud.com/t/topic/9083

https://bbs.fit2cloud.com/t/topic/9158

**Comment by wanghe-fit2cloud** [comment]
> 本机docker部署的postgresql 

提供一下部署方法

**Comment by jjscx** [comment]
docker run -d \
--name postgresql \
--restart always \
-p 5432:5432 \
-e POSTGRES_PASSWORD=xxxxxxx \
-e TZ=Asia/Shanghai \
-e PGDATA=/var/lib/postgresql/data/pgdata \
-v /data/postgresql:/var/lib/postgresql/data \
registry.cn-hangzhou.aliyuncs.com/xxxxx:postgresql

无其他操作 镜像是官方最新镜像未修改
________________________________
发件人: 王贺 ***@***.***>
发送时间: 2024年11月16日 10:42
收件人: 1Panel-dev/1Panel ***@***.***>
抄送: wtx ***@***.***>; Author ***@***.***>
主题: Re: [1Panel-dev/1Panel] [BUG] 1panel无法添加postgresql的远程服务器连接 (Issue #7091)


本机docker部署的postgresql

提供一下部署方法

―

**Comment by wanghe-fit2cloud** [comment]
使用你提供的命令，在本地服务器测试，复现不了问题，如下图：
<img width="1475" alt="image" src="https://github.com/user-attachments/assets/394f2a50-7767-4fa3-bb81-d3b3d37f9da3">

**Comment by tudan110** [comment]
账号是 root，无法添加；其他都可以
